tests/integ/test_workflow_with_clarify.py

```diff
@@ -0,0 +1,285 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import json
+import os
+import statistics
+import time
+import tempfile
+
+import pytest
+import numpy as np
+import pandas as pd
+from botocore.exceptions import WaiterError
+from sagemaker.amazon.linear_learner import LinearLearner, LinearLearnerPredictor
+from sagemaker.clarify import (
+    SageMakerClarifyProcessor,
+    BiasConfig,
+    DataConfig,
+    ModelConfig,
+    ModelPredictedLabelConfig,
```

Comment:
apply `isort` to imports:

```suggestion
    BiasConfig,
    DataConfig,
    ModelConfig,
    ModelPredictedLabelConfig,
    SageMakerClarifyProcessor,
```